### ISSUE TITLE
Some translation keys possibly not read

### ISSUE THREAD

**Issue by weidengeist** [kodolinija/stardeus-translations#129]
Hi @spajus,

I have noticed two keys in the translation files that are not used although they are present: `comfortable.work.spot` and  `slept.in.unfamiliar.bed`. The games uses the English terms instead.

Regards,
Weidengeist

**Comment by Ciizel** [comment]
Same problem with `nav.fail.being.immobilized`

![2022-10-22 13 14 22](https://user-images.githubusercontent.com/1397782/197318924-b4768f1a-e8a6-4da0-9562-e19e4618164b.png)

**Comment by spajus** [comment]
@weidengeist how did you figure those keys are not used? `slept.in.unfamiliar.bed` is surely used as a mood effect label, checked in code just now.

**Comment by weidengeist** [comment]
@spajus I saw it just yesterday in a stream on Twitch. The player had the most recent version and showed the current changelog in the beginning.
![screenshot10-22-12:01:16_annotated](https://user-images.githubusercontent.com/104255779/197333560-5a337f8f-a992-4974-9644-e7428d8f6846.jpg)

**Comment by weidengeist** [comment]
I have just taken a look into one of my own games and at least for `socialized.with` I can say that it **is** translated correctly. I will keep an eye out for the other keys mentioned above.

Could it be related to the Twitch streamer’s save file that it is not translated?

**Comment by spajus** [comment]
Yes, it is definitely going to be the save file keeping previous mood effect title. Translations are often cached in game state, since there would be a performance penalty to always translate everything.

**Comment by weidengeist** [comment]
Thank you very much for your answer. I will tell the Twitch streamer about that. I don’t consider it an issue that has to be resolved, because it will not matter in the final version when new games are created and the translations are complete.

**Comment by spajus** [comment]
There is a "clear history" button in the colonist mood list, that will keep only currently happening events, and those old translations will be wiped.
